dts/bindings/serial/zephyr,uart-bridge.yaml

```diff
@@ -0,0 +1,23 @@
+# Copyright 2025 Google LLC
+# SPDX-License-Identifier: Apache-2.0
+
+title: UART bridge
+
+description: |
+  Bridges data between two serial devices, for example a USB CDC-ACM serial
+  port and an hardware UART.
+
+  The device node must have exactly two child nodes for the two devices to be
+  bridged together. Example configuration:
```

Comment:
Just noticed this description is also out of date.

Reply:
Good catch thanks, still had the old schema in my head, had fixed the example forgot the description.